1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,11 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for (let name in salaries) {
+  if (salaries[name] > 50000) {
+    console.log(name + ": $" + salaries[name]);
```

Comment:
I think that these days, generally, you should use template literals where you would use `+`. So here: `name: $${salaries[name]}`